changelogs/fragments/2937-proxmox_kvm-change_default_values.yml

```
@@ -0,0 +1,4 @@
+---
+bugfixes:
```

These are not bugfixes, but breaking changes, and should be marked as such.
```suggestion
breaking_changes:
```

---

Changed in https://github.com/ansible-collections/community.general/pull/2937/commits/ceb5ec5da01fd9f36a12a1f122e6411562897c64